src/lib/support/logging/CHIPLogging.h

```diff
@@ -109,6 +109,7 @@ enum LogModule
     kLogModule_Shell,
     kLogModule_DeviceLayer,
     kLogModule_SetupPayload,
+    kLogModule_AdditionalDataPayload,
```

Comment:
This part is still (again?) here...

Reply:
i added it back because i needed to log inside of AdditionalDataPayloadParser.cpp but not anymore, removed it